src/Sylius/Component/Core/Model/Product.php

```diff
@@ -149,6 +149,7 @@ public function hasTaxon(TaxonInterface $taxon): bool
      */
     public function getChannels(): Collection
     {
+        /** @phpstan-ignore-next-line */
         return $this->channels;
```

Comment:
Why ignore?

Reply:
```
Method Sylius\Component\Core\Model\Product::getChannels() should return Doctrine\Common\Collections\Collection<(int|string),
         Sylius\Component\Channel\Model\ChannelInterface> but returns Doctrine\Common\Collections\Collection<(int|string), Sylius\Component\Core\Model\ChannelInterface>.
```
Obviously here it is right to return a list of core channels, it sounds like a PHPStan bug...